Loop/Managers/LoopAlertsManager.swift

```diff
@@ -50,7 +50,8 @@ public class LoopAlertsManager {
         let fgcontent = Alert.Content(title: title,
                                       body: fgBody,
                                       acknowledgeActionButtonLabel: NSLocalizedString("Dismiss", comment: "Default alert dismissal"))
-        alertManager?.issueAlert(Alert(identifier: bluetoothPoweredOffIdentifier, foregroundContent: fgcontent, backgroundContent: bgcontent, trigger: .immediate))
+        alertManager?.issueAlert(Alert(identifier: bluetoothPoweredOffIdentifier, foregroundContent: fgcontent, backgroundContent: bgcontent, trigger: .immediate,
+                                       interruptionLevel: .critical))
```

Comment:
[nit] Consider moving all to multiple lines, rather than just the last parameter. (Or collapse back to a single line.)